### PR TITLE
Add revocation documentation and graphics

### PR DESCRIPTION
**Pull request by rmlearney-digicatapult**
Adding revocation documentation to be merged after completion of functionality

### CODE REVIEW

File: demo/hyproof-demo.md [hunk 1]
```diff
@@ -141,5 +146,25 @@ The truth of this data could then be confirmed against the cryptographic data wi
 
 In this way `Heidi` and `Emma` keep each other honest, and the `Regulator` keeps them both honest.
 
+### 4. Revoking the certificate
```
Comment (dc-andysign) [inline demo/hyproof-demo.md:149]: Sorry , I can't point this to the right line but this is about line 114 .. from

4. `Emma` submits this embodied CO2 value to the ledger to be added to the certificate that `Heidi` previously **intitiated**


to 

4. `Emma` submits this embodied CO2 value to the ledger to be added to the certificate that `Heidi` previously **initiated**

File: demo/hyproof-demo.md [hunk 2]
```diff
@@ -141,5 +146,25 @@ The truth of this data could then be confirmed against the cryptographic data wi
...
+
+In order to keep `Reginald` themselves honest and auditable, the reasons for revocation must be recorded in an indelible, high integrity manner, and made available to others. This form asks `Reginald` to supply the reasons for revocation.
+
+![Revocation Form](./revocation_form.png)
+
+`Reginald` clicks submit, and the form is loaded into the system and attached to the relevant certificate.
+
+After a few seconds of processing the certificate is now `Revoked`. This is instantly visible to all.
+
+![Revoked Certificate](./revoked_certificate.png)
```
Comment (dc-andysign) [inline demo/hyproof-demo.md:165]: This is more like some sort of FYI comment .. I think I am supposed to remove the (OPEN ATTACHMENT) a link in the next PR but we can def keep the image as is as it is not that far away from the actual final thing.

**Review by dc-andysign — CHANGES_REQUESTED** [review]
Just one small typo ..

File: demo/hyproof-demo.md (hunk 2 above)
Comment (rmlearney-digicatapult) [reply demo/hyproof-demo.md:165]: I will update the docs once that's done

**Review by dc-andysign — APPROVED** [review]
👍

**Pull request merged**